app/views/dashboard/managedByAgentsProperties.scala.html

```diff
@@ -91,11 +91,7 @@ <h2 id="noAgentId" class="heading-secondary">
                         <td>
                             <ul class="list">
                                 <li>
-                                    <a href="@agentAppointment.routes.RevokeAgentController.revokeAgent(
-                                        p.submissionId,
-                                        p.agents.filter(ag =>
-                                            Some(ag.organisationId) == model.agentOrganisationId).head.authorisedPartyId,
-                                        model.agentCode)">
+                                    <a href="#">
```

Comment:
should this still be an anchor? maybe remove that?

Reply:
this page not used anymore, it is from when we had the old dashboard (when Fi was still in the team). I didn't remove it because I wanted this change to just be about the new agent relationship stuff and the issue we found the other day. We can remove this file (and the old Dashboard code) in another ticket. There is some redirect logic in the old Dashboard that we will need to handle/keep as well.

Reply:
well, it would hurt to remove the tags <a ...>...</a>

Reply:
but you have my thumb